osu.Game/Screens/OnlinePlay/Multiplayer/Spectate/PlayerGrid_Cell.cs

```diff
@@ -32,68 +31,79 @@ private partial class Cell : CompositeDrawable
             /// <summary>
             /// An action that toggles the maximisation state of this cell.
             /// </summary>
-            public Action<Cell> ToggleMaximisationState;
+            public Action<Cell>? ToggleMaximisationState;
 
             /// <summary>
             /// Whether this cell is currently maximised.
             /// </summary>
-            public bool IsMaximised;
+            public bool IsMaximised { get; private set; }
 
             private Facade facade;
-            private bool isTracking = true;
 
-            public Cell(int facadeIndex, Drawable content)
+            private bool isAnimating;
+
+            public Cell(int facadeIndex, Drawable content, Facade facade)
             {
                 FacadeIndex = facadeIndex;
+                this.facade = facade;
 
                 Origin = Anchor.Centre;
                 InternalChild = Content = content;
+
+                Masking = true;
+                CornerRadius = 5;
             }
 
             protected override void Update()
             {
                 base.Update();
 
-                if (isTracking)
-                {
-                    Position = getFinalPosition();
-                    Size = getFinalSize();
-                }
+                var targetPos = getFinalPosition();
```

Comment:
I had to switch away from using transforms because with the new animation style there's more going on. For example, when the facade is shrinking back to original size, the grid is scaling creating a moving target.